packages/components/src/core/InputDropdown/style.ts

```diff
@@ -290,19 +310,19 @@ const isDisabled = (props: InputDropdownProps): SerializedStyles => {
   `;
 };
 
-const doNotForwardProps = [
-  "intent",
-  "state",
-  "sdsStage",
-  "sdsType",
-  "isMinimal",
-  "shouldTruncateMinimalDetails",
-  "shouldPutAColonAfterLabel",
-];
-
-export const StyledInputDropdown = styled(Button, {
-  shouldForwardProp: (prop: string) => !doNotForwardProps.includes(prop),
-})`
+/**
+ * (masoudmanson): This is a temporary fix for the issue where the props
```

Comment:
Thanks for the comment! A tiny suggestion to emphasize the part of the temp fix 🙏 PTAL what you think thank you!

```suggestion
 * (masoudmanson):  `Button as React.ComponentType<InputDropdownProps>` is a temporary fix for the issue where the props
```